css/registrationForm.css

```diff
@@ -0,0 +1,23 @@
+form {
+  background-color: #f4f5f0;
+  width: 350px;
+  border: 1px groove;
+}
+h5 {
+  background-color: #97c00e;
+  text-align: center;
+  color: white;
+  padding: 15px;
+  margin: -1px -1px 20px -1px;
+}
+.userField {

```

Comment:
Nice PR. Pretty clean. If you would like to improve further you may want to read on some conventions, e.g.:  http://bdavidxyz.com/blog/how-to-name-css-classes/